app/linux/BUILD.gn

```diff
@@ -54,7 +54,10 @@ action("strip_and_dump_symbols") {
     brave_symbol_file = "$root_out_dir/brave.breakpad.$current_cpu"
   }
 
-  inputs = [ brave_binary, dump_syms_binary ]
+  inputs = [
+    #brave_binary,
```

Comment:
And if you want it commented out because it's temporary, documenting what needs to happen to re-enable is nice, even if it's just an issue link.